src/System.Private.ServiceModel/tests/Common/Infrastructure/OSHelper.cs

```diff
@@ -57,6 +53,18 @@ public static class OSHelper
             // So we can't determine the OS from the runtime in this list.
         };
 
+        // All Windows version currently use the runtime "win7" so cannot be distinguished by that.
+        // However the windows major and minor versions are known and can be used.
+        private static List<Tuple<string, OSID>> _descriptionToOSID = new List<Tuple<string, OSID>>
+        {
+            new Tuple<string, OSID>("Microsoft Windows 6.1.", OSID.Windows_7),
+            new Tuple<string, OSID>("Microsoft Windows 6.2.", OSID.Windows_8 | OSID.Windows_2008_R2),
+            new Tuple<string, OSID>("Microsoft Windows 6.3.", OSID.Windows_8_1),
+            new Tuple<string, OSID>("Microsoft Windows 10.", OSID.Windows_10),
+            new Tuple<string, OSID>("Microsoft Windows Phone", OSID.AnyWindows),
```

Comment:
@iamjasonp -- thanks -- those new enums have now been added and the review can continue.